client/gutenboarding/style.scss

```diff
@@ -1,11 +1,61 @@
 @import './gutenberg-styles/styles';
 
+// Override core variables: we don't have wp-admin header and sidebar in Calypso
+$admin-bar-height: 0;
+$admin-bar-height-big: 0;
+$admin-sidebar-width: 0;
+$admin-sidebar-width-big: 0;
+$admin-sidebar-width-collapsed: 0;
+
+// Calypso section wrapper resets
+.is-section-gutenboarding {
+	background: $white;
+
+	.layout__content {
+		margin: 0;
+		padding: 0;
+	}
+}
+
 .gutenboarding {
+	// Reset some Calypso base styles
```

Comment:
Would it make sense to import the resets from a separate file to keep concerns separate? (Thinking that if we have e.g. a `calypso-resets.scss` and a `gutenberg-resets.scss` file, people will be less tempted to scatter resets that they might need to add later all over `style.scss`.

Reply:
We can remove them with a separate entry-point approach (https://github.com/Automattic/wp-calypso/pull/37071) — meanwhile I think it's safe to keep just a couple lines here and remove them later on?

There might be some other overrides here that we can remove (or add more!) with an entry-point approach but I'd leave all that for a separate PR. This is already a good improvement.

Reply:
Okay, sounds good. Can we maybe add code comments (`@TODO`?) as to which resets we'll be able to remove with the separate-entrypoint PR?

Reply:
Added some notes/todos on https://github.com/Automattic/wp-calypso/pull/36943/commits/d67bb47c412b0bd1a2862c6c2a83fdf59ee22534